tools/RAiDER/delay.py

```diff
@@ -45,6 +45,10 @@ def tropo_delay(dt, weather_model_file, aoi, los, height_levels=None, out_proj=4
     out_proj: int,str           - (optional) EPSG code for output projection
     look_dir: str               - (optional) Satellite look direction. Only needed for slant delay calculation
     cube_spacing_m: int         - (optional) Horizontal spacing in meters when generating cubes
```

Comment:
What is the common practice for documentation generation in term of reporting default values?

Reply:
Not sure. planning to look more into documentation practice for next relase.